Systematics/test/splitTree.py

```diff
@@ -0,0 +1,150 @@
+import tree2dataset as t2d
```

Comment:
these script too should go into a subdirectory, actually if you put them under Systematics/scripts they will be available system wide trhough $PATH